### ISSUE TITLE
Request/Suggestion for editor support

### ISSUE THREAD

**Issue by zoedsoupe** [Matsuuu/pinkmare#6]
I would like to know if it’s possible to the project to also suport the [helix](https://github.com/helix-editor/helix) editor! It would be an awesome addition to the pallete of colorschemes.

Also great job here! Is one of my lovely themes!

**Comment by Matsuuu** [comment]
Heya! Do you happen to have any docs on how to implement one?

**Comment by zoedsoupe** [comment]
Yes of course! You can check the official helix docs https://docs.helix-editor.com/themes.html and also current themes implemented on the editor: https://github.com/helix-editor/helix/tree/master/runtime/themesZoey de Souza PessanhaDesenvolvedora de Softwarehttps://github.com/zoedsoupeOn 14 Nov 2023, at 03:59, Matsu ***@***.***> wrote:﻿
Heya! Do you happen to have any docs on how to implement one?
